src/mono/mono/mini/interp/mintops.def

```diff
@@ -705,6 +705,7 @@ OPDEF(MINT_MONO_NEWOBJ, "mono_newobj", 2, Pop0, Push1, MintOpClassToken)
 OPDEF(MINT_MONO_RETOBJ, "mono_retobj", 1, Pop1, Push0, MintOpNoArgs)
 OPDEF(MINT_MONO_ATOMIC_STORE_I4, "mono_atomic.store.i4", 1, Pop2, Push0, MintOpNoArgs)
 OPDEF(MINT_MONO_MEMORY_BARRIER, "mono_memory_barrier", 1, Pop0, Push0, MintOpNoArgs)
+OPDEF(MINT_MONO_EXCHANGE_LONG, "mono_interlocked_exchange_long", 1, Pop2, Push1, MintOpNoArgs)
 OPDEF(MINT_MONO_LDDOMAIN, "mono_lddomain", 1, Pop0, Push1, MintOpNoArgs)
```

Comment:
Should have a different name like EXCHANGE_I8, 'long' is a bit ambiguous.

Reply:
Are you suggesting just `EXCHANGE_I8` or `mono_interlocked_exchange_I8`?

Reply:
MINT_MONO_EXCHANGE_I8.

Reply:
Fixed.